tests/hmm/hmumu_utils.py

```diff
@@ -2925,7 +2943,13 @@ def create_datastructure(dataset_name, is_mc, dataset_era, do_fsr=False):
             ("Jet_area", "float32"),
             ("Jet_rawFactor", "float32")
         ],
-        "SoftActivityJet": [
+        #"FSRPhoton":[
```

Comment:
should this be enabled/disabled according to a flag (`do_fsr` or such)?

Reply:
This section is actually outdated, we now access the FSR Photon collection here :
https://github.com/irenedutta23/hepaccelerate-cms/blob/update_fsr_And_VBFsync/tests/hmm/hmumu_utils.py#L2984-L2995

Reply:
If needed, I can just remove those commented lines

Reply:
Aha, great, please remove them then!

Reply:
done!